topics/ecology/tutorials/openrefine_gbif/tutorial.md

```diff
@@ -0,0 +1,327 @@
+>
+> 1. Go to column "Full name" and click on then go to `Text facet`. Then click on `count`.
+>    - *Guzmania lingulata* is the first item in the list with 25 specimens, but it is also present in the 4th position with 20 specimens.
+> 2. Fix the error from the "Full name" column menu, `Edit Cells > Common transforms > Collapse consecutive whitespaces`.
+>
+{: .hands_on}
+
+> <question-title></question-title>
+>
+> 1. How many *Guzmania lingulata* records only appear in the list finally?
+>
+> > <solution-title></solution-title>
+> >
+> > 1. 45
+> >
+> {: .solution}
+>
+{: .question}
+
+> <hands-on-title> {% icon hands_on %} Hands-on: Faceting and duplicates. </hands-on-title>
+>
+> 1. Go to column catalog in "Cat. Numb", and follow the route `Facet > Customized facets > Duplicates facet`. 
+>    - The facet will show 4 duplicates
+> 2. Click on true, and you'll see the values in the main window
+> 3. After a check with the specimens labels, fix the values clicking edit directly on the cell with the correct catalogue numbers
+>    - UWP:122470 Vargas P
+>    - UWP:122471 Vargas I
+>    - UWP:157351 Betancur H
+>    - UWP:157339 Betancur J
+>
+{: .hands_on}
+
+## Filtering 
+
+> <hands-on-title> {% icon hands_on %} Hands-on: Basic filter. </hands-on-title>
+>
+> 1. Go again to "Full name" column menu and perform a `Text facet` to visualize the values, then go again to the column menu and click on `Text filter`, perform the following filters and fix them as described below:
+>    - search for "sp1" entries -> Then remove it and obtain "Cyperus", clicking on `Edit` directly in the cell
+>    - search for "SP2" entries, check `case sensitive` -> To remove it and obtain "Cyperus", you can `Edit` directly in the cell
+>    - search for "spp".
+>       - Go to "Full name" column menu, then click `Edit cells > Transform`
+>       - In the text box paste the formula `value.replace(" spp.", "")` and click `ok`
+>
+{: .hands_on}
+
+> <question-title></question-title>
+>
+> 1. Do you think you can use also `Edit cells > Transform` formula to apply the changes you made manually for the 2 first terms searched ("sp1" and "SP2")?
+> 2. Why you can easily edit directly these 2 first entries and not the third one?
+>
+> > <solution-title></solution-title>
+> >
+> > 1. Yes, using respectively `value.replace(" sp1", "")` and `value.replace(" SP2", "")` formulas
+> > 2. Because the 2 first searchs only concern one entry for each when the third one return several.
+> >
+> {: .solution}
+>
+{: .question}
+
+> <hands-on-title> {% icon hands_on %} Hands-on: Advance filter 1. </hands-on-title>
+>
+> 1. Go to column "genus" and perform a `Text filter`.
+> 2. Check `regular expression` and `case sensitive` , then paste the expression "^[a-z]"
+>    - This regular expression filters the strings in which the first letter is lowercase.
+> 3. Perform a correction since the genus should be capitalized.
```

Comment:
you could add our Regular Expression FAQ in this box if you like? https://training.galaxyproject.org/training-material/faqs/galaxy/analysis_regular_expressions.html